tests/radiation_transport/test_advective_transport.py

```diff
@@ -134,6 +164,37 @@ def test_plotting(self):
         ax = self.solver.plot(show=True)
         assert len(ax.lines) > 2
 
+    def test_analyse_SN(self, caplog):
```

Comment:
Not sure this is needed as the function is used within analyse for the single null class which this is within. However that should also be the case for the double null function using analyse within the double null class, which seemed to need the extra test.

Reply:
It may be overkill but we could refactor that at a later date but I'm not too worried right now